src/modules/link/model.rs

```diff
@@ -21,4 +20,8 @@ impl Record for Link {
     fn set_id(&mut self, id: &[u8]) {
         self.id = id.to_owned();
     }
+
+    fn set_updated_at(&mut self) {
```

Comment:
As `Repository.update` is introduced, we also need to ensure we can update the `updated_at` on `Record` instances